packages/flutter_driver/lib/src/driver/timeline_summary.dart

```diff
@@ -105,7 +108,10 @@ class TimelineSummary {
         .toList(),
       'frame_begin_times': _extractBeginTimestamps('Frame')
         .map<int>((Duration duration) => duration.inMicroseconds)
-        .toList()
+        .toList(),
+      'average_vsync_transitions_missed': sceneDisplayLagSummarizer.computeAverageVsyncTransitionsMissed(),
```

Comment:
You'll also need to add these keys to https://github.com/flutter/flutter/blob/19ff596979e407c484a32f4071420fca4f4c885f/dev/devicelab/lib/tasks/perf_tests.dart#L275 so they can show up in our cocoon dashboard.

Reply:
Done!